eidpauth/src/main/java/uk/ac/ucl/eidp/auth/AuthController.java

```diff
@@ -0,0 +1,81 @@
+package uk.ac.ucl.eidp.auth;
+
+import java.security.Principal;
+import java.util.Optional;
+import uk.ac.ucl.eidp.auth.jaas.SingleFactorCallbackHandler;
+import uk.ac.ucl.eidp.auth.jwt.Jwt;
+import uk.ac.ucl.eidp.auth.model.RoleE;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'uk\.ac\.ucl\.eidp\.auth\.model\.RoleE' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.